### PR TITLE
Update LAN Section, Namely Windows

### PR DESCRIPTION
**Pull request by kn0wmad**
Remove browser section, shift around mozilla apps

### CODE REVIEW

File: site/source/user-manual/connecting/connecting-lan/lan-ff.rst [hunk 1]
```diff
@@ -3,12 +3,9 @@
 ==================================
 Trusting Your Start9 CA in Firefox
 ==================================
+This guide applies to Firefox, Firefox ESR, and Librewolf.  Mozilla apps need to be configured to use the certrificate store of your device.  To find out why Mozilla does this differently, you can read their `blog post <https://blog.mozilla.org/security/2019/02/14/why-does-mozilla-maintain-our-own-root-certificate-store/>`_ on the topic.
```
Comment (elvece) [inline site/source/user-manual/connecting/connecting-lan/lan-ff.rst:6]:
```suggestion
This guide applies to Firefox, Firefox ESR, and Librewolf.  Mozilla apps need to be configured to use the certificate store of your device.  To find out why Mozilla does this differently, you can read their `blog post <https://blog.mozilla.org/security/2019/02/14/why-does-mozilla-maintain-our-own-root-certificate-store/>`_ on the topic.
```

File: site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst [hunk 2]
```diff
@@ -18,74 +18,78 @@ Unfortunately, Windows does not have mDNS alias support built-in, which is neces
...
+#. Verify your server's unique `<adjective-noun> Local Root CA` certificate is in the “Certificates” folder:
 
-   .. figure:: /_static/images/ssl/windows/8_windows_successful_cert_install.png
-    :width: 90%
+   .. figure:: /_static/images/ssl/windows/10_successful_cert_install.png
+    :width: 50%
     :alt: Successful cert install
 
-    Your server's unique `<custom-address> Local Root CA` is now imported into Certificate folder
+#. You can save the console settings (where we added a snap-in) if desired.  The CA certificate will remain imported to the CA certificate store either way, and you wil likely use this guide if you need to import a new certificate.
```
Comment (elvece) [inline site/source/user-manual/connecting/connecting-lan/lan-os/lan-windows.rst:89]:
```suggestion
#. You can save the console settings (where we added a snap-in), if desired.  The CA certificate will remain imported to the CA certificate store either way, and you will likely use this guide if you need to import a new certificate.
```

**Review by elvece — CHANGES_REQUESTED** [review]
Few typos, added suggestions.

**Comment by kn0wmad** [comment]
> Few typos, added suggestions.

Committed, thank  you

**Pull request merged**